src/actions/actions.ts

```diff
@@ -265,10 +265,10 @@ export const AppActions = {
       },
     };
   },
-  updateKeyboardHeight: (height: number) => {
+  updateKeyboardSize: (width: number, height: number) => {
     return {
-      type: APP_UPDATE_KEYBOARD_HEIGHT,
-      value: height,
+      type: APP_UPDATE_KEYBOARD_SIZE,
+      value: { height, width },
```

Comment:
```suggestion
      value: { width, height },
```

Of course, your code works, but I think that the pair "width" and "height" should be lined up as "width, height".